airbyte-migration/src/main/java/io/airbyte/migrate/migrations/MigrationV0_11_0.java

```diff
@@ -46,7 +46,7 @@
 
   @Override
   public String getVersion() {
-    return "v0.11.0-alpha";
+    return "0.10.0-alpha";
```

Comment:
Why is this down to 0.10.0 when the class is V0_11_0?

Reply:
getVersion and the class name should match. that's a mistake. as for the version itself, that's going to to be fluid until we "officially" release the migration stuff.